webpack.config.js

```diff
@@ -2,7 +2,7 @@ var path = require('path');
 var webpack = require('webpack');
 
 module.exports = {
-  devtool: 'eval',
+  devtool: 'cheap-module-eval-source-map',
```

Comment:
Breakpoints (appear to) work with `-eval`, but only if they're not sync. Chrome only loads the source maps after it finishes running the `eval` call itself.